### PR TITLE
tweak: add more slow query basic fields

### PR DESCRIPTION
**Pull request by shhdgit**
image

### CODE REVIEW

**Comment by shhdgit** [comment]
Merged first for the 12.02 release.

File: ui/packages/tidb-dashboard-lib/src/apps/SlowQuery/translations/zh.yaml [hunk 1]
```diff
@@ -21,6 +21,12 @@ slow_query:
     is_internal: 是否为内部 SQL 查询
     is_success: 是否执行成功
     is_success_tooltip: SQL 查询是否执行成功
+    is_prepared: Is Prepared?
```
Comment (lilyjazz) [inline ui/packages/tidb-dashboard-lib/src/apps/SlowQuery/translations/zh.yaml:24]: 是否执行准备？

File: ui/packages/tidb-dashboard-lib/src/apps/SlowQuery/translations/zh.yaml [hunk 2]
```diff
@@ -21,6 +21,12 @@ slow_query:
...
+    is_prepared_tooltip: 是否由 prepare 语句生成
+    is_plan_from_cache: Is Plan from Cache?
```
Comment (lilyjazz) [inline ui/packages/tidb-dashboard-lib/src/apps/SlowQuery/translations/zh.yaml:26]: 是否使用计划缓存？

File: ui/packages/tidb-dashboard-lib/src/apps/SlowQuery/translations/zh.yaml [hunk 3]
```diff
@@ -21,6 +21,12 @@ slow_query:
...
+    is_plan_from_cache_tooltip: 查询计划是否来自缓存
+    is_plan_from_binding: Is Plan from Binding?
```
Comment (lilyjazz) [inline ui/packages/tidb-dashboard-lib/src/apps/SlowQuery/translations/zh.yaml:28]: 是否绑定执行计划？

Comment (shhdgit) [reply ui/packages/tidb-dashboard-lib/src/apps/SlowQuery/translations/zh.yaml:28]: https://github.com/pingcap/tidb-dashboard/commit/d0228dda6a5a0551d1395bc4f99183be8ed8166f

Here~

**Pull request merged**